.github/workflows/pr.yml

```diff
@@ -2,11 +2,17 @@ name: Validate PR
 
 on:
   pull_request:
-    branches: [ master ]
+    branches: [master]
     types: [assigned, opened, synchronize, labeled]
 
+# Using env variables for step conditions to pass if vars are not found.
```

Comment:
I'm not sure why env vars needed to be introduced just to work into conditionals.

Reply:
me neither, other than it throws saying it can't find secrets.****